### PR TITLE
Update ColorRect class ref documentation

### PR DESCRIPTION
**Pull request by mrawlingst**
Added new info and examples to ClassRect class ref doc.

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -12541,14 +12541,22 @@
 </class>
 <class name="ColorRect" inherits="Control" category="Core">
 	<brief_description>
+	Colored rect for canvas.
```
Comment (akien-mga) [inline doc/base/classes.xml]: Missing tab indent here.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -12541,14 +12541,22 @@
...
 	</brief_description>
 	<description>
+	An object that is represented on the canvas as a rect with color. [Color] is used to set or get color info for the rect.
```
Comment (akien-mga) [inline doc/base/classes.xml]: Here too.

File: doc/base/classes.xml [hunk 3]
```diff
@@ -12541,14 +12541,22 @@
...
 	</description>
 	<methods>
 		<method name="get_frame_color" qualifiers="const">
 			<return type="Color">
 			</return>
 			<description>
+			Return the color in RGBA format.
```
Comment (akien-mga) [inline doc/base/classes.xml]: Also missing tab indentation here. Also, the empty newline before the codeblock is not necessary, each line is a paragraph in the XML.

**Comment by mrawlingst** [comment]
Updated - I didn't realize I didn't indent them. Whoops!

**Pull request merged**